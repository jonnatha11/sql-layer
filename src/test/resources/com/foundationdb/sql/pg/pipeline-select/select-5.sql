SELECT * FROM artists WHERE (
 (artists.id NOT IN
    (SELECT join_albums_artists.artist_id FROM join_albums_artists
        INNER JOIN albums ON (albums.id = join_albums_artists.album_id)
        INNER JOIN join_albums_artists AS albums_artists_0 ON (albums_artists_0.album_id = albums.id)
        WHERE ((albums_artists_0.artist_id = 1) AND (join_albums_artists.artist_id IS NOT NULL))))
 OR (artists.id IS NULL))