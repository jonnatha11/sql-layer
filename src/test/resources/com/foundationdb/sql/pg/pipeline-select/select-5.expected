id	name
3	c
